<launch>

  <node name="pose_estimate_vicon" pkg="pose_estimate_vicon" type="pose_estimate_vicon_node" respawn="false" output="screen">
	<param name="input_topic" value="/kinect2/rgb_rect/image"/>
	<param name="smallObj" value="true"/>
	<param name="output_topic" value="/camera/image/objectPose"/>
	<param name="output_topic_test" value="/camera/image/test"/>
	<param name="threshold_for_homography" value="0.3"/>
	<param name="param1" value="560"/>
	<param name="param2" value="320"/>
	<!--param name="param3" value="640"/-->
	<param name="param3" value="1155"/>
	<param name="param4" value="735"/>
	<param name="original_image" value="/home/ir/perception_ws/src/object_perception/data/kinect2_obj01.png"/>
	<param name="contour_image" value="/home/ir/perception_ws/src/object_perception/data/kinect2_contour01.png"/>
	<param name="createData" value="true"/>
    <param name="camera_calib_file" value="/home/ir/kinect2WS/src/iai_kinect2/kinect2_bridge/data/507554242542/calib_color.yaml"/>
    <param name="vicon_marker_name" value="arnab_object_01_0%d"/>
    <param name="target_frame" value="/vicon/arnab_small_object/arnab_small_object" />
    <param name="threshold_vicon_error" value="0.5" />
  </node>
</launch>
